pkgs/development/python-modules/indexed_zstd/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
```

There's a name space collision with zstd
```suggestion
, buildPythonPackage
, pkgs
```